wrappers/python/pyrs_sensor.cpp

```diff
@@ -85,25 +86,30 @@ void init_sensor(py::module &m) {
     roi_sensor.def(py::init<rs2::sensor>(), "sensor"_a)
         .def("set_region_of_interest", &rs2::roi_sensor::set_region_of_interest, "roi"_a) // No docstring in C++
         .def("get_region_of_interest", &rs2::roi_sensor::get_region_of_interest) // No docstring in C++
-        .def("__nonzero__", &rs2::roi_sensor::operator bool); // No docstring in C++
+        .def("__nonzero__", &rs2::roi_sensor::operator bool)  // Called to implement truth value testing in Python 2
+        .def("__bool__", &rs2::roi_sensor::operator bool);    // Called to implement truth value testing in Python 3
 
     py::class_<rs2::depth_sensor, rs2::sensor> depth_sensor(m, "depth_sensor"); // No docstring in C++
     depth_sensor.def(py::init<rs2::sensor>(), "sensor"_a)
         .def("get_depth_scale", &rs2::depth_sensor::get_depth_scale,
              "Retrieves mapping between the units of the depth image and meters.")
-        .def("__nonzero__", &rs2::depth_sensor::operator bool); // No docstring in C++
+        .def("__nonzero__", &rs2::depth_sensor::operator bool) // Called to implement truth value testing in Python 2
```

Comment:
For device, it's enough to have these function in the base class -- will it work if we don't repeat this for every derivation?

Reply:
In sensor, yes correct.
will be removed